module/Photo/src/Photo/Service/Metadata.php

```diff
@@ -21,7 +21,7 @@ class Metadata extends AbstractService
      */
     public function populateMetadata($photo, $path)
     {
-        $exif = read_exif_data($path, 'EXIF');
+        $exif = exif_read_data($path, 'EXIF');
```

Comment:
Interesting. Didn't know that `read_exif_data` has an alias.

Reply:
`read_exif_data` is the alias of `exif_read_data`